eventmesh-common/build.gradle

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-def grpcVersion = '1.64.0'
+
 
 dependencies {
```

Comment:
There is no need to leave a blank line, which would result in three blank lines separating the first line of source code from the beginning.